### DISCUSSION TITLE
Scrum Prep Milestone 3-2

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#183] [General]
Each team member will comment progress made through the past week.

**Comment by YampierFernandez** [comment]
Worked on task to connect the users model to the backend.

**Comment by AlvinBadillo** [comment]
This week I did some research on what task need to be created for the db.

**Comment by ActuallySeb** [comment]
Studying for exams

**Comment by JuanDelPueblo** [comment]
This week I created and began working on a task to move the old sign-up/login logic to the new components.

**Comment by JuanIranzo** [comment]
This week I could not work a lot on the project. However, I created a documentation task to start revamping and integrating feedback to the Milestone 3 Documentation

**Comment by cristian-barreras** [comment]
I was not able to work on project during the week. I will be working on it during the weekend.

**Comment by Cabinetman** [comment]
Mostly research for the database routes and Docker.

**Comment by Christian2101** [comment]
Started reading the professors feedback of the documentation to see what needs to be fixed.

**Comment by PerezMarcelo** [comment]
Currently working on enhancing the whole website and cleaning up the code by removing any unnecessary documents.

**Comment by elliot-gomez** [comment]
This week, I focused on reviewing the oncoming tasks and ongoing discussions, but I didn’t get much done through this busy election week.
- Elliot J. Gomez Coriano

**Comment by CdoxHype** [comment]
This week I focused on learning new skills to prepare for the Web Frontend Redesign.

-Julio E Camacho

**Comment by gcal77** [comment]
This week I did research for upcoming tasks for the web dev design. I couldn't do much else because of the busy election week.

- Gabriel Calvente

**Comment by Kaariinaa08** [comment]
This week I opened a Lecture Topic Task that focused on seeing the hypothetical errors we might come across when we launch our website and how we can overcome them.

- Karina Lopez Rodriguez
